spec.html

```diff
@@ -30862,6 +30869,21 @@
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-get-regexp.prototype.dotAll">
+        <h1>get RegExp.prototype.dotAll</h1>
```

Comment:
Nit: The rest of the methods are arranged in alphabetical order. If you want to continue that with this patch, this could go a little higher in the file.

Reply:
Good idea. Done.